src/Common/config.h.in

```diff
@@ -59,6 +59,7 @@
 #cmakedefine01 USE_PRQL
 #cmakedefine01 USE_OPENSSL_INTREE
 #cmakedefine01 USE_ULID
+#cmakedefine01 USE_SEASONAL
```

Comment:
The preprocessor symbols are needed only for 3rd party libraries. Since you technically wrote your own STL implementation, l. 62 can be deleted.